website_product_supplier/__openerp__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    'name': "Website Product Supplier",
+    'category': 'Website',
+    'version': '8.0.1.0.0',
+    'depends': [
+        'auth_supplier',
+        'website_portal',
+        'product_manufacturer'
```

Comment:
@rafaelbn And if i add more than one supplier, i have more than one owner for the same product and this is not correct.